pkg/cli/set/triggers.go

```diff
@@ -60,14 +60,14 @@ var (
 		trigger for a build config will only trigger the first build.`)
 
 	triggersExample = templates.Examples(`
-		# Print the triggers on the registry
-	  %[1]s triggers dc/registry
+		# Print the triggers on the deployment 'myapp'
```

Comment:
Can you fix the indentation here, while at it, also this says `deployment`, but should `deployment config`. We need to be extra careful not to confuse users.

Reply:
done